webpack.config.js

```diff
@@ -19,7 +19,7 @@ const config = {
   },
   plugins: [
     new HtmlWebpackPlugin({
-      template: "index.html",
+      template: "invite_your_team.html",
```

Comment:
I made a webpack config for multiple files in #66, please, fetch upstream, test it out and then everyone will have to change one line in `webpack.config.js`